LICENSE

```diff
@@ -1,642 +1,301 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
+		    GNU GENERAL PUBLIC LICENSE
+		       Version 2, June 1991
```

Comment:
Please do not change the license of the package.
One has been chosen, we should now keep it.

Reply:
Even if it's not the one in the upstream package file ?
* https://github.com/leenooks/phpLDAPadmin/blob/master/LICENSE

Reply:
This license file is for the package itself, not the upstream app.

Reply:
My bad, I haven't understood the difference between the package license and the upstream application license. I use to use the same for both :/